lib/split/helper.rb

```diff
@@ -9,15 +9,16 @@ def ab_test(metric_descriptor, control = nil, *alternatives)
       begin
         experiment = ExperimentCatalog.find_or_initialize(metric_descriptor, control, *alternatives)
         alternative = if Split.configuration.enabled
-          experiment.save
-          trial = Trial.new(:user => ab_user, :experiment => experiment,
-              :override => override_alternative(experiment.name), :exclude => exclude_visitor?,
-              :disabled => split_generically_disabled?)
-          alt = trial.choose!(self)
-          alt ? alt.name : nil
-        else
-          control_variable(experiment.control)
-        end
+                        experiment.version = Split::Experiment.version_for_experiment_key(ab_user.key_for_experiment(experiment)).to_i
+                        experiment.save
+                        trial = Trial.new(:user => ab_user, :experiment => experiment,
+                                          :override => override_alternative(experiment.name), :exclude => exclude_visitor?,
+                                          :disabled => split_generically_disabled?, :version => Split::Experiment.version_for_experiment_key(ab_user.key_for_experiment(experiment)).to_i)
+                        alt = trial.choose!(self)
+                        alt ? alt.name : nil
+                      else
+                        control_variable(experiment.control)
+                      end
       rescue Errno::ECONNREFUSED, Redis::BaseError, SocketError => e
```

Comment:
It's hard to understand what's changed w/the white space changes. Can that be submitted in a separate PR?

Reply:
I'll just reset the whitespace for now